docs/quickstart.rst

```diff
@@ -47,29 +47,30 @@ Using ProxyStore
 ++++++++++++++++
 
 Colmena can use `ProxyStore <https://github.com/gpauloski/ProxyStore>`_ to efficiently transfer large objects, typically on the order of 100KB or larger, between the thinker and workers directly.
-To enable the use of ProxyStore, a ProxyStore backend must be initialized.
-The name of the ProxyStore backend and a threshold value (bytes) can be passed via the parameters :code:`proxystore_name` and :code:`proxystore_threshold` to :code:`make_queue_pairs`.
+To enable the use of ProxyStore, a `Store <https://docs.proxystore.dev/main/api/store/base/#proxystore.store.base.Store>`_ must be initialized.
```

Comment:
Could we switch this from passive to active voice? E.g., 

"Enable proxy store by first initializing a `Store <...>` then passing the name and threshold size for the store to `make_queue_pairs`.`